deployment/stacks/api.ts

```diff
@@ -15,6 +15,7 @@ import {
 import { Lambda } from "../constructs/lambda";
 import { WafConstruct } from "../constructs/waf";
 import { addIamPropertiesToBucketAutoDeleteRole } from "../utils/s3";
+import { getSubnets } from "../utils/vpc";
```

Comment:
Shouldn't the VPC info and the subnet info come into the stack and "travel" together?  Meaning VPC name and subnet IDs.

Reply:
I could make it a more complicated nested thing but the vpc name isn't necessary to get the subnets if I have the ids. I really don't care unless we're going to do some nifty checks to see if the subnets are in the vpc but since we are configuring I think we should trust ourselves. If we went a name based filtering of subnets inside the vpc I'd be more inclined to.

Reply:
We huddled and changes were made. 🙏